src/PhpBrew/VariantBuilder.php

```diff
@@ -315,34 +315,27 @@ public function __construct()
             return;
         };
 
-        $this->variants['curl'] = function (Build $build, $prefix = null) {
-            if ($prefix) {
```

Comment:
Why is this condition removed? What's the point of looking up the prefix if it's passed explicitly?

Reply:
I did not remove it, I just follow the `libxml2` you modified, and change the name to `$val`. And move it to the following statements
```php
            if ($build->compareVersion('7.4') < 0) {
                if ($val) {
                    return "--with-curl=$val";
                }
            ...
```

Reply:
Please see the big comment below.